examples/js/controls/OrbitControls.js

```diff
@@ -423,6 +426,17 @@ THREE.OrbitControls = function ( object, domElement ) {
 
 	}
 
+
+	function isPanEvent( event ) {
+
+		return scope.panKeys.some( function ( key ) {
```

Comment:
Sry, for being unclear. But the idea is not to create a function for each call of Array.some.

Reply:
```
function isPanEvent( event ) {
	var hasKey = function ( key ) {
		return event[ key ];
	};
	return scope.panKeys.some( hasKey );
}
```
is it so good?

Reply:
👍

Reply:
👍